cli_options/test/cli_options_test.exs

```diff
@@ -297,6 +297,23 @@ defmodule CliOptionsTest do
                CliOptions.parse(["--file", "foo.ex,bar.ex"], schema)
     end
 
+    test "multiple with default set" do
+      schema = [file: [type: :string, multiple: true, default: ["foo.ex", "bar.ex"]]]
+
+      assert {:ok, {[file: ["foo.ex", "bar.ex"]], [], []}} = CliOptions.parse([], schema)
+
+      # `default` does not have to be a list if `multiple` is true
+      schema = [file: [type: :string, multiple: true, default: "foo.ex"]]
+
+      assert {:ok, {[file: ["foo.ex"]], [], []}} = CliOptions.parse([], schema)
+
+      # multiple works as expected
+      assert {:ok, {[file: ["bar.ex"]], [], []}} = CliOptions.parse(["--file", "bar.ex"], schema)
+
+      assert {:ok, {[file: ["foo.ex", "bar.ex"]], [], []}} =
```

Comment:
can we also add a check with `:multiple` set to `true` and and invalid value for default?